### PR TITLE
Personal profile page created in builders folder

### PR DESCRIPTION
**Pull request by temitopeohassan**
## Description

I created a personal page in the builders folder
profilepage

## Additional Information

- [ ] I have read the [contributing docs](/scaffold-eth/scaffold-eth-2/blob/main/CONTRIBUTING.md) (if this is your first contribution)
- [ ] This is not a duplicate of any [existing pull request](https://github.com/scaffold-eth/scaffold-eth-2/pulls)

## Related Issues
 related to issue #8 
_Closes #{issue number}_

Your ENS/address:
0x2b2542b313385AF7453623765383CE3cc5D3c9cF

### CODE REVIEW

**Comment by derrekcoleman** [comment]
Thanks for creating the builders folder, @temitopeohassan! I like that you keep the intro focused and professional - the Memoji picture is a nice touch.

I encourage you to practice giving your bio's styling a little more attention. The bio section is currently a series of `li` in an `ul`. Try using tailwind classes to create spacing between elements or wrapping the bio section in a box, maybe with a rounded outline or shadow.

If you'd like inspiration, we often use [DaisyUI components](https://daisyui.com/components/) here in SE-2 - you could use other tailwind-based component libraries, such as [Shadcn](ui.shadcn.com/docs/components/) as well. Check out components like Card, Tooltip, or anything that catches your eye. This is a great chance to try out new things and develop your taste.

**Comment by temitopeohassan** [comment]
@derrekcoleman thank you for the feedback! I am new to Tailwind, i've mostly used styled components, bootstrap and material ui for most of my developments up till now. I will work on the suggestions you made

File: package.json [hunk 1]
```diff
@@ -37,6 +37,9 @@
   },
   "packageManager": "yarn@3.2.3",
   "devDependencies": {
+    "@types/cli-progress": "^3.11.5",
+    "@types/concat-stream": "^2.0.3",
+    "@types/parse-json": "^4.0.2",
```
Comment (technophile-04) [inline package.json:42]: Lets remove this since we are not using them

File: packages/nextjs/app/page.tsx
Comment (technophile-04) [inline packages/nextjs/app/page.tsx:1]: Lets revert this changes, not related to this PR ?

File: packages/nextjs/app/builders/0x2b2542b313385AF7453623765383CE3cc5D3c9cF/page.tsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  return (
+    <>
+      <div className="flex-grow bg-base-300 w-full mt-16 px-8 py-12">
+        <h1 className="text-center">
+          <span className="block text-2xl mb-2">My </span>
+          <span className="block text-4xl font-bold">Profile</span>
+        </h1>
+        <div className="card card-side bg-base-100 shadow-xl">
+          <figure>
+            <img src="/profileimage.jpeg" alt="Avatar" />
```
Comment (technophile-04) [inline packages/nextjs/app/builders/0x2b2542b313385AF7453623765383CE3cc5D3c9cF/page.tsx:16]: Lets use Next Image component since we have the image, if not the gh actions fails https://github.com/BuidlGuidl/batch5.buidlguidl.com/actions/runs/9046061959/job/24885136673?pr=14

File: packages/nextjs/app/builders/0x2b2542b313385AF7453623765383CE3cc5D3c9cF/page.tsx [hunk 3]
```diff
@@ -0,0 +1,43 @@
+"use client";
```
Comment (technophile-04) [inline packages/nextjs/app/builders/0x2b2542b313385AF7453623765383CE3cc5D3c9cF/page.tsx:1]: We don't need "use client", checkout https://github.com/BuidlGuidl/batch5.buidlguidl.com/pull/15#discussion_r1597931094

File: yarn.lock
Comment (technophile-04) [inline yarn.lock:1]: Lets revert this changes, 

After removing https://github.com/BuidlGuidl/batch5.buidlguidl.com/pull/14/files#r1597987212 and running `yarn install` this changes will be automatically reverted

**Review by technophile-04 — COMMENTED** [review]
This is looking great @temitopeohassan !! Just added some comments, could you resolve them ? Thanks 🙌

File: packages/nextjs/app/builders/0x2b2542b313385AF7453623765383CE3cc5D3c9cF/page.tsx [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+  return (
+    <>
+      <div className="flex-grow bg-base-300 w-full mt-16 px-8 py-12">
+        <h1 className="text-center">
+          <span className="block text-2xl mb-2">My </span>
+          <span className="block text-4xl font-bold">Profile</span>
+        </h1>
+        <div className="card card-side bg-base-100 shadow-xl">
+          <figure>
+            <Image src="/assets/profileimage.jpeg" alt="Temitope O Hassan" width={399} height={399} />
```
Comment (technophile-04) [inline packages/nextjs/app/builders/0x2b2542b313385AF7453623765383CE3cc5D3c9cF/page.tsx:14]: Screenshot 2024-05-14 at 4 16 58 PM


I think you forgot to update the correct path, it should be `assets/temitopeohassan.jpeg`  ?

Comment (temitopeohassan) [reply packages/nextjs/app/builders/0x2b2542b313385AF7453623765383CE3cc5D3c9cF/page.tsx]: I have corrected the error on the page

**Review by technophile-04 — APPROVED** [review]
Thanks @temitopeohassan  ! Looking great !
